src/python/turicreate/test/test_object_detector.py

```diff
@@ -313,6 +329,21 @@ def test_evaluate_missing_annotations(self):
             del sf[self.annotations]
             self.model.evaluate(sf.head())
 
+    def test_evaluate_with_missing_annotations_label(self):
+
+        def create_missing_annotations_label(x):
+            for y in x:
+                y['label'] = None
+            return x
```

Comment:
well, 2 newlines are recommended as the interval between function the other codes.

Reply:
good